starknet-accounts/tests/single_owner_account.rs

```diff
@@ -32,7 +32,7 @@ fn create_sequencer_client() -> SequencerGatewayProvider {
 
 fn create_jsonrpc_client() -> JsonRpcClient<HttpTransport> {
     let rpc_url = std::env::var("STARKNET_RPC")
-        .unwrap_or("https://pathfinder.rpc.sepolia.starknet.rs/rpc/v0_7".into());
+        .unwrap_or_else(|_| "https://pathfinder.rpc.sepolia.starknet.rs/rpc/v0_7".into());
```

Comment:
I don't know which lint this is but this seems an overhead no? As the previous approach was already good

Reply:
I will revert to earlier implementation, although the overhead is negligible for creating closure of a string.
Using unwrap_or_else can also generally be better as it's default value now can be evaluated lazily(computed only if needed), in case of string, performance gain is negligible and overhead minmal.